sdk/src/main/java/com/hcaptcha/sdk/HCaptchaException.java

```diff
@@ -2,36 +2,48 @@
 
 import androidx.annotation.Nullable;
 
-import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.NonNull;
 
 
 /**
  * A checked exception which contains an {@link HCaptchaError} id and message.
  */
 @Data
-@AllArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class HCaptchaException extends Exception {
 
     /**
      * The hCaptcha error object
      */
-    private final HCaptchaError hCaptchaError;
+    @NonNull
+    private final HCaptchaError error;
+
+    @Nullable
+    private final HCaptchaRetryer retryer;
+
+    public HCaptchaException(HCaptchaError error) {
+        this(error, null);
+    }
+
+    HCaptchaException(@NonNull HCaptchaError error, @Nullable HCaptchaRetryer retryer) {
+        this.error = error;
+        this.retryer = retryer;
+    }
 
     /**
      * @return The {@link HCaptchaError} error
      */
-    public HCaptchaError getHCaptchaError() {
-        return this.hCaptchaError;
+    public HCaptchaError getError() {
```

Comment:
This introduces breaking change, doesn't it?

Reply:
Good catch